pkg/restic/common.go

```diff
@@ -39,13 +39,18 @@ const (
 	InitContainer               = "restic-wait"
 	DefaultMaintenanceFrequency = 24 * time.Hour
 
-	podAnnotationPrefix       = "snapshot.velero.io/"
+	// Deprecated.
+	podAnnotationPrefix = "snapshot.velero.io/"
+
+	// Deprecated.
```

Comment:
We're not deprecating this one, at least not as part of this PR.  This one indicates which volumes the user **wants** to back up.